src/pystatis/table.py

```diff
@@ -172,9 +182,9 @@ def parse_regio_table(data: pd.DataFrame) -> pd.DataFrame:
 
         # Extracts new column names from first values of the Merkmal_Label columns
         # and assigns these to the relevant code columns (Auspraegung_Code)
-        codes = data.filter(like="Auspraegung_Code")
-        codes.columns = data.filter(like="Merkmal_Label").iloc[0].tolist()
-        codes.columns = [code + "_Code" for code in codes.columns]
+        # codes = data.filter(like="Auspraegung_Code")
```

Comment:
Remove?

Reply:
that is mentioned in the PR description: Jonas added this part to have the codes that he needed to join the data with external data. I think we should not do this for Regionalstatistik only but for all methods and give the user a parameter to decide so I left this for better reference...however we have git so maybe its better to go to a previous commit instead of having some commented code that is not used so I will remove it here